pkg/antctl/raw/set/command.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2020 Antrea Authors
```

Comment:
Date is wrong. Didn't you get an error from CI?

Reply:
Fixed. I didn't get an error...